bot/cogs/math.py

```diff
@@ -1,21 +1,48 @@
 #!/usr/bin/env python3
 # -*- coding: utf-8 -*-
 
-
+import asyncio
+import sys
+from contextlib import suppress
 from io import BytesIO
 from re import search
-
+from subprocess import PIPE, Popen, STDOUT, TimeoutExpired  # noqa: B404
```

Comment:
This comment isn't required, since we aren't using Bandit to blacklist this import statement.